src/Action/Smtp.hs

```diff
@@ -98,6 +104,9 @@ sendMailToAddressIO logger receiver msg = do
         Left (e :: IOException) ->
             throwSendMailError $ IOErrorRunningSendMail e
   where
+    subjectLabel = case msg ^. msgSubjectLabel of
+        IdeaSpaceSubject is -> cs $ showIdeaSpace is
+        UserLoginSubject ul -> ul ^. unUserLogin
```

Comment:
two throughts (no need to act on them):

1. I think like `where` only if it is used in a function that has a very small body (3 lines at most) and does everything in the `where` block.  In this case, I prefer `let`, because it lets the author write things down in the order in which they need to be read in order to be understood.
1. I would have instantiated `HasUILabel` with `EmailSubjectLabel`, but arguably this is the only place where it's ever needed, so I'm not sure about this any more.